lib/irgen.ml

```diff
@@ -45,6 +45,7 @@ let translate (binds, sfuncs): L.llmodule =
   in
 
   let build_func_body fn = 
+    let (the_func, _) = StringMap.find fn.sfname all_funcs in
```

Comment:
was going to make this change in my PR, nice!

Reply:
yessirrrr